bin/travis-ci/conf.vexpress_ca15_tc2_qemu

```diff
@@ -21,7 +21,7 @@
 console_impl=qemu
 qemu_machine="vexpress-a15"
 qemu_binary="qemu-system-arm"
-qemu_extra_args="-nographic -m 1G -tftp /tftpboot"
+qemu_extra_args="-nographic -m 1G -tftp ${UBOOT_TRAVIS_BUILD_DIR}"
```

Comment:
Yes, this is also another copy/paste example from my local setup.  Telling QEMU to add a network device is more along the lines of "Add _this_ type of thing _here_" rather than "add a sensible network device" sadly.